### PR TITLE
WEBAPP-326: fix error handling

### PR DESCRIPTION
**Pull request by Schedulaar**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/endpoint/__tests__/Endpoint.spec.js [hunk 1]
```diff
@@ -93,8 +92,7 @@ describe('Endpoint', () => {
         stateName,
         'invalid json response body at undefined reason: Unexpected token I in JSON at position 0'
       )
-      const loadingError = new LoadingError({endpointName: stateName, message: mappingError.message})
-      const payload = new Payload(false, defaultMapParamsToUrl(params), null, loadingError)
+      const payload = new Payload(false, defaultMapParamsToUrl(params), null, mappingError)
```
[the remark this answers is not in the document]
Comment (maxammann) [reply src/modules/endpoint/__tests__/Endpoint.spec.js]: Maybe add a test --coverage run to the Jenkinsfile.

[the remark this answers is not in the document]
Comment (Schedulaar) [reply src/modules/endpoint/__tests__/Endpoint.spec.js]: Yep, runs without problems... Should we also try to run jest without the --runInBand flag?

[the remark this answers is not in the document]
Comment (maxammann) [reply src/modules/endpoint/__tests__/Endpoint.spec.js]: We can try, yes. 

About the coverage bug: We just run both for now I think. It's probably a bug in jest but I can not find any information.

File: Jenkinsfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ pipeline {
         sh 'yarn'
         sh 'yarn run flow'
         sh 'yarn run lint'
-        sh 'yarn run test --runInBand'
+        sh 'yarn run test --coverage'
```
Comment (maxammann) [inline Jenkinsfile]: We should run both, because they behave differently

Comment (Schedulaar) [reply Jenkinsfile]: Done

**Pull request merged**